docs/README.md

```diff
@@ -591,6 +591,8 @@ curl -d 'measurement,tag1=value1,tag2=value2 field1=123,field2=1.23' -X POST 'ht
 
 </div>
 
+Please, note that VictoriaMetrics also exposes endpoint for InfluxDB v2 HTTP API at `/influx/api/v2/write` and `/api/v2/write`.
```

Comment:
should we provide the curl example how to ingest? and what will be retrieved

Reply:
Format there is the same, has more to do with hardcoded `v2` prefix in Influx V2 and the fact that we don't mentioned that it is supported anywhere in doc

Reply:
anyway it worth to mention, otherwise user must go and check the section for v1

Reply:
Updated, ptal